pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -118,23 +118,18 @@ func IsBackupScheduled(backup *Backup) bool {
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
-// IsBackupClean returns true if a Backup has been successfully cleaned up
-func IsBackupClean(backup *Backup) bool {
+// IsBackupCleanedUp returns true if a Backup has been successfully cleaned up
+func IsBackupCleanedUp(backup *Backup) bool {
 	_, condition := GetBackupCondition(&backup.Status, BackupClean)
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
-// IsCleanCandidate returns true if a Backup should be added to clean candidate according to cleanPolicy
-func IsCleanCandidate(backup *Backup) bool {
-	switch backup.Spec.CleanPolicy {
-	case CleanPolicyTypeDelete, CleanPolicyTypeOnFailure:
-		return true
-	default:
-		return false
-	}
+// CleanPolicyIsNotRetain returns true if backup's clean policy is not CleanPolicyTypeRetain
+func CleanPolicyIsNotRetain(backup *Backup) bool {
+	return backup.Spec.CleanPolicy != CleanPolicyTypeRetain
 }
 
-// NeedNotClean returns true if a Backup need not to be cleaned up according to cleanPolicy
-func NeedNotClean(backup *Backup) bool {
+// DontCleanSuccessBackup returns true if a Backup need not to be cleaned up according to cleanPolicy
+func DontCleanSuccessBackup(backup *Backup) bool {
```

Comment:
It's better not showing `not` in the function name.

Reply:
BTW, can we just define two new functions like `shouldClean()` and `shouldKeep()` or something similar?